package.json

```diff
@@ -75,6 +75,7 @@
     "socket.io-client": "1.7.3",
     "stylelint": "7.9.0",
     "urijs": "1.18.9",
-    "webpack": "2.2.1"
+    "webpack": "2.2.1",
+    "webpack-remove-debug": "0.1.0"
```

Comment:
This comes from https://github.com/johngodley/webpack-remove-debug.
1 commit only, 0 stars/forks, pretty much no downloads... Shouldn't we inline the [6 lines of code](https://github.com/johngodley/webpack-remove-debug/blob/master/index.js) of this project directly into our Webpack config, instead?

Also, I'm curious of such little usage of this, what do other projects do, they just leave the debug stuff in there?
Finally, do we want to remove this from both production and development environment, or is there an interest in keeping this in dev mode?